script/Ciel/Day 7/07_01_CIEL07_2_1A.txt

```diff
@@ -1,23 +1,23 @@
 [sha:d155a0f3e2e78032ab2d2a5c68ceeaa849c3823c]{
 -- Page 10009, Offset 15212.
 -- 「わたしの方に来るとはまったく予想外でした。
-"I didn't expect you to come to me.
+"I have to say I'm rather surprised you chose to come to me.
 }
 [sha:96ef3f0e4cfacc6dda1a7266cd966f3ae76477ff]{
 -- Page 10009, Offset 15213.
 -- 遠野くん的には、ノエル先生の方が話しやすいんじゃないですか？」
-Wouldn't you consider Noel-sensei to be easier to talk to?"
+I thought you found Noel-sensei a little easier to talk to?"
 }
 [sha:cf1e46406a79ea170c01ab6f5bf02bedd7746216]{
 -- Page 173, Offset 15214.
 -- いつになく刺々しい先輩だった。
-Senpai is being uncharacteristically snappy.
+Senpai is being uncharacteristically snippy.
 }
 [sha:b40dbaac4ae6803386e26d571048cce7c77efc96]{
 -- Page 173, Offset 15215.
 -- 今朝のノエル先生との会話……いや、さかのぼって瓦礫に埋まっていた時のコトを責められている気がする……。
-I get the feeling she's still condemning me for my conversation with Noel this morning――actually,
- maybe even going back to the time we were buried together under the rubble.
+I get the sense that she's been eyeing me with reproach ever since the conversation with Noel-sensei this morning――
```

Comment:
prefer 'my conversation'

Reply:
"my conversation" doesn't really sound particularly natural given Ciel was there too. "the conversation" sounds better